src/server/clients_info.rs

```diff
@@ -28,19 +34,45 @@ pub(crate) struct ClientBuffers {
 }
 
 impl ClientsInfo {
+    pub(super) fn new(policy: VisibilityPolicy) -> Self {
+        Self {
+            info: Default::default(),
+            policy,
+        }
+    }
+
+    /// Returns reference to connected client info.
+    ///
+    /// This operation is *O*(*n*).
+    pub fn get(&self, client_id: ClientId) -> Option<&ClientInfo> {
```

Comment:
I think most of the time people will write `get(client_id).unwrap()`. Maybe it worth to provide or replace with a panicking API with a nice message?

Reply:
I mean that I doubt that anyone need to write logic like `if let Some(client_info) = clients_info.get(client_id) { // Some logic } else { // Something else }`

Reply:
If you agree about the panicking versions, I will go only with them for now because I hardly imagine a use cases where `Option` versions can be used. If someone will request it - I will be happy to add them.

Reply:
Never mind, just found a use-case - for events.

Reply:
Yep, I pushed both methods yesterday.